notebook/static/notebook/js/quickhelp.js

```diff
@@ -235,41 +249,21 @@ define([
 
     QuickHelp.prototype.build_command_help = function () {
         var command_shortcuts = this.keyboard_manager.command_shortcuts.help();
-        return build_div('<h4>Command Mode (press <code>Esc</code> to enable)</h4>', command_shortcuts);
-    };
-
-    var special_case = { pageup: "PageUp", pagedown: "Page Down", 'minus': '-' };
```

Comment:
this is now baked into `humanize_sequence`.  The logic was convoluted and wrong.